controller.lua

```diff
@@ -210,22 +210,6 @@ function cntrl:humanBytes(bytes)
     return math.floor(result * mult + 0.5) / mult.." "..size[factor+1]
 end
 
--- format a timestamp to date
-function cntrl:formatDate(ts)
-    return os.date('%Y-%m-%d %H:%M:%S', ts)
-end
-
--- urlencode a string
-function cntrl:urlEncode(str)
-    if (str) then
-        str = string.gsub (str, "\n", "\r\n")
-        str = string.gsub (str, "([^%w ])",
-        function (c) return string.format ("%%%02X", string.byte(c)) end)
-        str = string.gsub (str, " ", "+")
-    end
-    return str
-end
```

Comment:
This function is duplicated, see [controller.lua:244](https://github.com/clandmeter/aports-turbo/blob/master/controller.lua#L244).